blindsign/blindrsa/blindrsa_test.go

```diff
@@ -362,7 +362,7 @@ func verifyTestVector(t *testing.T, vector testVector) {
 }
 
 func TestVectors(t *testing.T) {
-	data, err := ioutil.ReadFile("test_vectors.json")
+	data, err := ioutil.ReadFile("testdata/test_vectors.json")
```

Comment:
Note for reviewer: this file indeed got moved.

Reply:
I created a single commit for this change